docs/Reference/jatos.js-Reference.md

```diff
@@ -1296,7 +1297,7 @@ JSON Patch test operation: Tests that the specified value is set in the document
 
 ### `jatos.batchSession.add`
 
-JSON Patch add operation: Adds a value to an object or inserts it into an array. In the case of an array, the value is inserted before the given index. The `-` character can be used instead of an index to insert at the end of an array (see [jsonpatch.com](http://jsonpatch.com/)). If the path already exists in the Batch Session the value will be overwritten.
+JSON Patch add operation: Adds a value to an object or inserts it into an array. In the case of an array, the value is inserted before the given index. The `-` character can be used instead of an index to insert at the end of an array (see [jsonpatch.com](http://jsonpatch.com/)). If the path already exists in the Batch Session the value will be overwritten. The patch will fail if a key other than the innermost one does not already exist.
```

Comment:
> The patch will fail if a key other than the innermost one does not already exist.
I'm a bit lost what you mean. Can you give an example?

Reply:
I'm trying to say that `add("/a/b/c")` only works if `find("/a/b")` is defined. I want to document it because it surprised me, I thought it would create the intermediate objects, but possibly there is a better way to phrase it.

[This addition](https://github.com/JATOS/JATOS_docs/pull/17/commits/5f407662256d1cf100ce9280d5dd581d8ca1bf10#diff-6ea2ebbc4a486684b17b954e055e089af78cfa5aaa49a7459d94d1db46ffe0f5R1347) makes clear what it means. Maybe I should just move it to the first paragraph.

Reply:
How about this instead of 'The patch will fail if a key other than the innermost one does not already exist.':

A patch will fail if other than the last path element is missing, e.g. if, in the path `/a/b/c` any of `a` or `b` do not already exist before the add, the patch will fail.

Reply:
How about this instead of 'The patch will fail if a key other than the innermost one does not already exist.':

A patch will fail if other than the last path element is missing, e.g. if, in the path `/a/b/c` any of `a` or `b` do not already exist before the add, the patch will fail.

Reply:
How about this instead of 'The patch will fail if a key other than the innermost one does not already exist.':

A patch will fail if other than the last path element is missing, e.g. if, in the path `/a/b/c` any of `a` or `b` do not already exist before the add, the patch will fail.

Reply:
How about this instead of 'The patch will fail if a key other than the innermost one does not already exist.':

A patch will fail if other than the last path element is missing, e.g. if, in the path `/a/b/c` any of `a` or `b` do not already exist before the add, the patch will fail.